standards/SPIFFE_Workload_API.md

```diff
@@ -158,6 +158,12 @@ message X509SVID {
 
     // Required. ASN.1 DER encoded X.509 bundle for the trust domain.
     bytes bundle = 4;
+
```

Comment:
We should also reflect this change in the workloadapi.proto file in this directory.